src/Components/LoginForm/index.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useCallback, useContext, useState } from 'react';
+
+import { ErrorBanner, AuthInput, Button } from '../';
+import { AuthContext, validateInput } from '../../utils';
+
+import styles from './styles.module.scss';
+
+export const LoginForm = () => {
+
+    const { isUserAuthorized, setAuthStatus } = useContext(AuthContext);
+
+    const [formData, setFormData] = useState({
+        nameFiled: {
+          fieldValue: '',
+          isFieldTouched: false,
+          isFieldCorrect: false,
+          isFieldEmpty: true,
+          invalidCredentialsError: false
+        },
+        passwordField: {
+          fieldValue: '',
+          isFieldTouched: false,
+          isFieldCorrect: false,
+          isFieldEmpty: true,
+          isFieldPassValidation: false
+        },
+        invalidCredentialsError: false
+      }
+    );
+
+    const handleLoginChange = useCallback((value) => {
+      setFormData((prevState) => ({
+        ...prevState,
+        nameFiled: {
+          ...prevState.nameFiled,
+          fieldValue: value
+        },
+        invalidCredentialsError: false
+      }));
+    }, []);
+
+    const handlePasswordChange = (value) => {
```

Comment:
I think these duplication can be refactored and some `useInput` hook introduced to cover all the input in your app